test/playwright/utils/navigation.ts

```diff
@@ -90,7 +121,100 @@ export const currentContentType = async (page: Page) => {
 }
 
 export const dismissTranslationBanner = async (page: Page) => {
-  await page
-    .locator('[data-testid="banner-translation-ar"] [aria-label="Close"]')
-    .click()
+  await page.locator('[data-testid="banner-translation"] button').click()
+}
+
+export const goToSearchTerm = async (
+  page: Page,
+  term: string,
+  searchType: SearchType = ALL_MEDIA,
+  options: {
+    mode?: RenderMode
+    dir?: 'ltr' | 'rtl'
+    query?: string // Only for SSR mode
+  } = {}
+) => {
+  const dir = options.dir || 'ltr'
+  const mode = options.mode ?? 'SSR'
+  const query = options.query ? `&${options.query}` : ''
```

Comment:
It seems like maybe `searchType` should also go in `options` to make it so someone could easily do a CSR of the all content search. Otherwise they either have to pass ALL_MEDIA or undefined to coerce the default out.

```suggestion
  options: {
    searchType?: SearchType
    mode?: RenderMode
    dir?: 'ltr' | 'rtl'
    query?: string // Only for SSR mode
  } = {}
) => {
  const searchType = options.searchType ?? ALL_MEDIA
  const dir = options.dir || 'ltr'
  const mode = options.mode ?? 'SSR'
  const query = options.query ? `&${options.query}` : ''
```

Reply:
Thank you! This will make calling the helper function much easier :)